crates/client/assets/schema.sdl

```diff
@@ -850,6 +850,7 @@ type Query {
 	messages(owner: Address, first: Int, after: String, last: Int, before: String): MessageConnection!
 	messageProof(transactionId: TransactionId!, nonce: Nonce!, commitBlockId: BlockId, commitBlockHeight: U32): MessageProof
 	messageStatus(nonce: Nonce!): MessageStatus!
+	status(id: Bytes32!): RelayedTransactionStatus
```

Comment:
Would be nice to use a GraphQL scalar that wraps the RelayedTransactionId type. Using more specific newtypes/scalars helps with typescript parsing of queries.

Reply:
Oh yeah. Thanks for the reminder. I tried that and it wouldn't compile for some reason, so I just went with`Bytes32` to keep moving. I can try again though.